examples/plot_intelex.py

```diff
@@ -0,0 +1,273 @@
+"""
+Creating models that are accelerated by Intel(R) Extension for scikit-learn
+--------------------------------------------------------------------------------
+
+Introduction
+============
+
+This guide demonstrates how Intel(R) Extension for scikit-learn (also
+``scikit-learn-intelex``, or ``sklearnex``) can be used to speed up inference of
+Scikit-learn models.
+
+The extension supports most of Scikit-learn's classical machine learning
+algorithms, like k-nearest neighbors, support vector machines, linear/logistic
+regression, and more. Stock Scikit-learn implementations are used where no
+optimized version is available, making this package 100% compatible with
+existing code. An up-to-date list of supported algorithms can be found in the
+`official documentation
+<https://intel.github.io/scikit-learn-intelex/algorithms.html>`_.
```

Comment:
Is there any check in sklearnex that a patched model produces the exact same output? So e.g. can I assume that `np.allclose(clf.predict_proba(X), clf_opt.predict_proba(X))`? Or is it more like the algorithm is the same but results can be different, e.g. due to random initialization? If it's the former, it would be nice to mention that.

Reply:
sklearnex's backend is completely separate from scikit-learn, including a separate RNG. So there's no guarantee that identical outputs are achieved and, in fact, they're mostly different.
The agreement of model outputs is however tested in our CI https://github.com/intel/scikit-learn-intelex/blob/master/tests/test_examples.py

I'll mention this in the example